### DISCUSSION TITLE
Autoplay inputs failure

### DISCUSSION THREAD

**Discussion by iijj22gg** [ppy/osu#27907] [Q&A]
### Type

Game behaviour

### Bug description

I was messing around with the new renderers on Linux (that's basically all I do with my computer, mess around with stuff and be terrible at games), when I opened a map with autoplay and noticed some weird behavior.

First, the HUD overlay did not appear, which I usually have set to "Hide During Gameplay". Then, I noticed that, although the cursor was circling around the spinner, it did not appear to be spinning, nor did a clear indicator appear, nor did the speed rise from zero. Then the notes appeared, there were no hitsounds, and the autoplay failed. (I have all hit indicators turned off/invisible in my skin.) It appeared as if the autoplay was not actually hitting the notes for some reason.

After a few tries, I tried switching renderers to see if the renderer was the cause of the issue, but I was unable to replicate it afterwards.

I've never seen this before, nor did I see anything online or in the issues about it (or I could be dumb and have searched the wrong thing), but I think it would be best to document this in case it happens again.

I attached the runtime log, that should have all the info you might want. Interesting stuff in there.

### Version

2024.412.1

### Logs

[1713240701.runtime.log](https://github.com/ppy/osu/files/14986984/1713240701.runtime.log)

**Comment by Walavouchey** [comment]
i suggest you post in https://github.com/ppy/osu/discussions/27659 and check the survey questions regarding linux/vulkan there

**Comment by iijj22gg** [comment]
I'm not sure is actually related to the Vulkan renderer, or I would have posted in #27659

**Comment by murryguy** [comment]
Are you using Vulkan?
